### PR TITLE
Auto assign title field

### PR DESCRIPTION
**Pull request by actlikewill**
- This adds the auto assign title flag, to control whether the system generates a title for judgments

### CODE REVIEW

**Comment by actlikewill** [comment]
@longhotsummer this is how I propose we should control whether to assign titles in the judgments, to allow KL to retain titles coming from Tausi

File: peachjam/models/judgment.py [hunk 1]
```diff
@@ -261,6 +261,13 @@ class Judgment(CoreDocument):
     )
     hearing_date = models.DateField(null=True, blank=True)
 
+    auto_assign_title = models.BooleanField(
+        help_text=_("Whether or not the system should assign the title"),
+        null=True,
+        blank=True,
```
Comment (longhotsummer) [inline peachjam/models/judgment.py:267]: drop both of these, so it must be either True or False, with a default of True. It'll backfill with True.

File: peachjam/models/judgment.py [hunk 2]
```diff
@@ -261,6 +261,13 @@ class Judgment(CoreDocument):
     )
     hearing_date = models.DateField(null=True, blank=True)
 
+    auto_assign_title = models.BooleanField(
```
Comment (longhotsummer) [inline peachjam/models/judgment.py:264]: it needs a translatable field title, "Auto-assign title"

**Comment by longhotsummer** [comment]
looks good, simplest way for now.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**